### ISSUE TITLE
Prendre en compte la date de naissance pour la question de durée de travail de l'ASS

### ISSUE THREAD

**Issue by MattiSG** [sgmap/mes-aides-ui#368]
En tant que bénéficiaire potentiel demandeur d'emploi né il y a moins de 26 ans,
Je ne vois une question « avez-vous travaillé au moins 5 ans dans les 10 dernières années » que si elle est pertinente au vu de mon âge,
Pour accélérer la saisie des données

- - - - - -

## Communication

- [ ] Mail du 28/08/2016 à 00:53.

**Comment by fpagnoux** [comment]
À regarder de plus près, c'est peut-être pas le seul problème qu'on ait avec cette question...
![image](https://cloud.githubusercontent.com/assets/11834997/18045619/feade5c6-6dd3-11e6-937d-3e12e869ea9c.png)

**Comment by MattiSG** [comment]
+1 mail du 3 oct. 2016 à 09:21

On donne toujours une plage de dates entre j-20 ans et j-10 ans.

**Comment by fpagnoux** [comment]
Moins de 26 ans bof: si j'ai 22 ans et que je travaille depuis mes 17 ans, je remplis la condition. Moins de 21 ans ?

**Comment by fpagnoux** [comment]
Le premier cas rapporté par mail semble lié au bug ou à une faute de saisie: la personne indique être née en 1988 et avoir terminé son dernier contrat de travail en 2001.

La meilleure règle serait: si j'avais moins de 21 ans lorsque mon dernier contrat de travail s'est arrêté, alors on ne me pose pas la question supplémentaire et on suppose que non, je n'ai pas pu travailler 5 ans dans les 10 années qui ont précédées.

Cela raccourcit la saisie des personnes dont le dernier contrat de travail s'est terminée lorsqu'ils avaient entre 16 et 21 ans.

**Comment by MattiSG** [comment]
> la personne indique être née en 1988 et avoir terminé son dernier contrat de travail en 2001.

Non, justement, c'est le système qui lui dit toujours « 2001 », comme dans la capture d'écran plus haut.

> si j'avais moins de 21 ans lorsque mon dernier contrat de travail s'est arrêté

:+1:

Il me semble surtout que cette issue est moins importante que #379, et qu'il s'agissait en premier lieu d'une mauvaise qualification.
